### PR TITLE
 Add the icons next to layer properties labels tabs

### PR DESCRIPTION
**Pull request by DelazJ**
because in layer styling panel only icons are used, no label.

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1132,7 +1132,8 @@ sharing easier.
 Formatting tab
 ..............
 
-In the :guilabel:`Formatting` tab, you can define a character for a line break
+In the |labelformatting| :guilabel:`Formatting` tab, you can define a character
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Is it possible to make the image a bit smaller?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1174,8 +1175,8 @@ transparent labels) will allow you to create outlined text labels.
 Background tab
 ..............
 
-In the :guilabel:`Background` tab, you can define with :guilabel:`Size X` and
-:guilabel:`Size Y` the shape of your background.
+In the |labelbackground| :guilabel:`Background` tab, you can define with
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Is it possible to make the image a bit smaller?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -1193,8 +1194,8 @@ map canvas using the :guilabel:`Blend mode` (see :ref:`blend-modes`).
 Shadow tab
 ..........
 
-Use the :guilabel:`Shadow` tab for a user-defined :guilabel:`Drop shadow`.
-The drawing of the background is very variable.
+Use the |labelshadow| :guilabel:`Shadow` tab for a user-defined :guilabel:`Draw
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Is it possible to make the image a bit smaller?

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 4]
```diff
@@ -1227,7 +1228,7 @@ can also be altered by choosing a blend mode.
 Placement tab
 .............
 
-Choose the :guilabel:`Placement` tab for configuring label placement
+Choose the |labelplacement| :guilabel:`Placement` tab for configuring label placement
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Is it possible to make the image a bit smaller?

**Comment by DelazJ** [comment]
Thanks @havatv for your reviews.
I'll update the images size though I wonder how the rendering is managed between GH preview and html (see for example opentable and formview [here](https://github.com/qgis/QGIS-Documentation/blob/bc20a254f0703a54525a3288d4a7929211ca6008/source/docs/documentation_guidelines/substitutions.rst#attribute-table) and [there](https://docs.qgis.org/testing/en/docs/user_manual/working_with_vector/attribute_table.html#table-view-vs-form-view))

**Comment by havatv** [comment]
Could the width attribute be used for (re)sizing?

> .. |labelplacement| image:: /static/common/labelplacement.png
>    :width: 1.5em

**Comment by DelazJ** [comment]
I think this is a github preview issue. browsing the docs in github, there's no relation between icons size rendering here and in the html version; the width property will if I'm not wrong rescale it in html.

**Pull request merged**